Makefile

```diff
@@ -86,7 +86,7 @@ vendor:
 run: gen
 	go build -o $(NOOBAA_OPERATOR_LOCAL) -mod=vendor $(CMD_MANAGER)
 	$(BIN)/noobaa-operator-local operator run
-	.PHONY: run
+.PHONY: gen
```

Comment:
why is it `.PHONY: gen` and not `.PHONY: run`?

Reply:
I misread the job . I correct it

Reply:
@v-harihar
Sorry, you are correct, my mistake 😄
I thought that the target is gen... and the target is run...
it should actually be as you wrote, `.PHONY: run`